designs/administrator/olm/List-of-operators-in-all-projects/list-of-operators.md

```diff
@@ -0,0 +1,41 @@
+
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+
+- When a user has selected the All Projects view from the context selector on the Installed Operators page, here they can view all of their Installed Operators. The goal of this design is to simplify how a user views the namespaces and associated operator(s) when they have selected to view the Installed Operators under All Projects.
+
+- When a user clicks the link in the table, “View 8…”, an anchor link takes the user to the Namespaces in the body of the Operator Details page. In this flow, we opt for a filterable experience in which the user can later filter by namespace or project under tabs such as Provided APIs or Events. (Scroll to view experience)
+
+- In an alternate path, if a user were to click into the Operator itself, it would take the user to the top of the Operator Details page rather than the anchor link in the body of the page. 
+
+![list-of-operators](img/list-of-operators.jpg)
+
+## Operator Details Page
+
+
+Here you can see the Namespaces in the body of the Operator Details page:
```

Comment:
Might be worth highlighting that the project selector is still 'all-projects', which in my opinion is one of the most novel parts of this design. This is one of the first times in the console (aside from API Explorer) that the user can view an object in a "global" context using the project selector, with the ability to still view it in the context of a single-namespace (by using the new kebab action in the namespace table or by manually changing the project selector.